### DISCUSSION TITLE
Feb 20

### DISCUSSION THREAD

**Discussion by miniposi** [GymHubCommunity/GymHub-FE#190] [DailyMeeting]
### 개발 현황
슬희: persigned url 하다가 잤음~~ 회원 탈퇴
수: 운동 기록 마무리
수영: textarea에 해시태그 넣기 
윤혁: 게시글 조회, 디테일 (?), 무한스크롤 적용

footer 떼는 법 ...
react query 다들 쓰고 있쪼!?

### 🏀 오늘 할 일

위와 같음

시간이 없다 ..

### 🎱 TMI

슬희: 햄버거에 빠짐 🍔 롯데리아 핫크리스피 버거 3개 먹을 예정 (티렉스 버거) **바스 버거** 추천
수: 손에 상처 나면 여드름 패치를 붙이자!! 
수영: 스타벅스 갈 예정 ☕ 서민들은 오지마라?
윤혁: 보험 비가 너무 비싸다 😭 스타벅스 나도 따라간다

**Comment by hanseulhee** [comment]
아자아자 !!
